lib/ruby_lsp/requests/completion.rb

```diff
@@ -36,7 +36,7 @@ class << self
         def provider
           Interface::CompletionOptions.new(
             resolve_provider: true,
-            trigger_characters: ["/", "\"", "'", ":", "@", "."],
+            trigger_characters: ["/", "\"", "'", ":", "@", ".", "="],
```

Comment:
To delegate requests properly, the server needs to register for the expected characters for delegation. For HTML attributes, we want to show completion after the equals.

Reply:
If I understand this correctly, should it include `<` for tag name completion? Seems to work if I add it and feels better.

Reply:
`*` as well for emmet abbreviations like `div*5`, though that doesn't work for some reason. Adding `"*"` does get one character further (the `*`) but when typing the number it disappears. Doesn't seem to work just like that.

They can start with a few characters (`.` for class, `#` for id), and maybe more.

https://code.visualstudio.com/docs/editor/emmet
https://docs.emmet.io/abbreviations/